src/frontend/src/lib/stores/address.store.ts

```diff
@@ -3,19 +3,17 @@ import type { Address, BtcAddress, EthAddress } from '$lib/types/address';
 import type { CertifiedData } from '$lib/types/store';
 import { writable, type Readable } from 'svelte/store';
 
-type AddressData<T extends Address = Address> = CertifiedData<T>;
+type AddressData<T extends Address> = CertifiedData<T>;
```

Comment:
Verbose but, should this be actually renamed tp `CertifiedAddressData` and the other option as `StorageAddressData`?

Somehow I'm a bit confuse about which types is what. WDYT?

Reply:
yes, make sense. doing it in this PR